docs/reporting.rst

```diff
@@ -0,0 +1,164 @@
+
+* **Kappa/Rho Scatter:** This is a scatter plot of `Kappa` vs. `Rho` features for all components.
+  In the plot, each dot represents a different component. The x-axis represents the kappa feature, and the
+  y-axis represents the rho feature. These are two of the most
+  informative features describing the likelihood of the component
+  being BOLD or non-BOLD. Additional information is provided via color
+  and size. In particuale, color informs about its classification
+  status (e.g., accepted, rejected); while size relates to
+  the amount of variance explained by the component (larger dot,
+  larger variance).
+.. image:: /_static/rep01_kapparhoScatter.png
+  :align: center
+  :height: 400px
+
+* **Kappa Scree Plot:** This scree plots provides a view of the components ranked by `Kappa`
+  As in the previous plot, each dot represents a component. The color of the dot inform us
+  about classification status. In this plot size is not related to variance explained.
+.. image:: /_static/rep01_kappaScree.png
+  :align: center
+  :height: 400px
+
+* **Rho Scree Plot:** This scree plots provides a view of the components ranked by `Rho`
+  As in the previous plot, each dot represents a component. The color of the dot inform us
+  about classification status. In this plot size is not related to variance explained.
+.. image:: /_static/rep01_rhoScree.png
+  :align: center
+  :height: 400px
+
+* **Variance Explained Plot:** This pie plot provides a summary of how much variance is explained
+  by each individual component, as well as, the total variance explained by each of the three
+  classification categories (i.e., accepted, rejected, ignored). In this plot, each component is
+  represented as a wedge, whose size is directly related to the amount of variance explained. The
+  color of the wedge inform us about the classification status of the component. For this view,
+  components are sorted by classification first, and inside each classification group by variance
+  explained.
+.. image:: /_static/rep01_varexpPie.png
+  :align: center
+  :height: 400px
+
+Individual Component View 
+-------------------------
+This view provides detailed information about an individual 
+component (selected in the summary view, see below). It includes three different
+interactive plots.
+
+* **Time series:** This plot shows the time series associated with a given component
+  (selected in the summary view). The x-axis represents time (in units of TR), and the
+  y-axis represents signal levels (in arbitrary units). Finally, the color of the trace
+  informs us about the component classification status.
+
+.. image:: /_static/rep01_tsPlot.png
+  :align: center
+  :height: 150px
+
+* **Spectrum:** This plot shows the spectrogram associated with a given component
+  (selected in the summary view). The x-axis represents frequency (in Hz), and the
+  y-axis represents spectral amplitude? Finally, the color of the trace
```

Comment:
Is this a question for the reviewer or something that had to be checked but was not?

Reply:
? and . are close to each other in the American keyboard. So, it's probably just a typo.